File: Arrays/two_sum.py
#Basically given an array of numbers and you are told to find two numbers that add up to a given target, then return the indices of those two numbers 

nums=[3,2,4]

def two_sum(array, target):
    
    indices_list=[]
    
    for i in range(len(array)):
        
        for j in range(i+1, len(nums)):
            
            if array[i] + array[j] == target:
                
                indices_list.append(i)
                
                indices_list.append(j)
                
    return indices_list

File: Arrays/twoSum.js
//In this leetcode problem, you are given a particular array and a target, you browse through all the elements of the array and check if they are any two elements that can add up to give you the target, if so you return the indexes else you return an empty array

let nums=[2,5,5,11];

function twoSum(arrayName, target){

    for(let i=0; i<arrayName.length; i++){

        for(let j=i+1; j<arrayName.length; j++){

            if(arrayName[i] + arrayName[j] === target){

                return[i,j];
            }
        }
    }
    return [];
}

console.log(twoSum(nums, 10));

//for this problem the mistake that i was making is that i started j at 1, instead of j=i+1